client/components/popover/menu-item.jsx

```diff
@@ -14,14 +28,21 @@ var MenuItem = React.createClass( {
 	},
 
 	render: function() {
-		var onMouseOver = this.props.focusOnHover ? this._onMouseOver : null;
+		const onMouseOver = this.props.focusOnHover ? this._onMouseOver : null;
+
+		let icon;
+		if ( this.props.icon ) {
```

Comment:
I tend to like conditional renders in the place where they render if they are not extremely complex :)